src/azul/bigquery_reservation.py

```diff
@@ -0,0 +1,140 @@
+from typing import (
+    Optional,
+)
+
+import attr
+from google.cloud import (
+    bigquery_reservation_v1 as bigquery_reservation,
+)
```

Comment:
```suggestion
from google.cloud.bigquery_reservation_v1 import (
    ReservationServiceClient,
    CapacityCommitment,
    Reservation,
    Assignment,
)
```

Changing the import will allow to use the classes directly in function signatures, which I think is a bit more consistent with Azul.